cli/app.go

```diff
@@ -762,6 +768,7 @@ var app = &cli.App{
 							Name: trainFlagJobStatus,
 							Usage: "training status to filter for. can be one of " +
 								"[unspecified, pending, in_progress, completed, failed, canceled, canceling]",
+							Required: true,
```

Comment:
Took a look at the code; I think this flag should be required.

Reply:
wish these options were enums... but totally beyond the scope of this PR

Reply:
Ah yeah I wondered the same thing; it looks like [training statuses _are_ enums at the proto level](https://github.com/viamrobotics/api/blob/main/proto/viam/app/mltraining/v1/ml_training.proto#L68-L76) and therefore are a `map[string]int32` in the generated Go code?

Reply:
interesting, yeah we could potentially surface those values - just need to strip `TRAINING_STATUS_`, downcase, and have some stable sorting - it's awkward but could be helpful.

that said, the internal command `dataListTrainingJobs` does return a validation error against that map so we have at least some guardrails, so I'm happy to leave as is if you prefer.

Reply:
I tried to add an enum (iota-const-block) but it ended up being a little messy with the existing `mltrainingpb.TrainingStatus_value` "enum". I added an `allTrainingStatusValues` function that gives all possible values for the flag instead of hardcoding it in the `trainFlagJobStatus` `Usage` text. Let me know what you think. Usage text still looks like:
```
--job-status value  training status to filter for. can be one of [unspecified, pending, in_progress, completed, failed, canceled, canceling]
```

Reply:
nice! do these appear in a stable order?

Reply:
Nope 😵‍💫 ; good catch! `slices.Sort`ed the entries so they should be consistent on each print.